pilot/pkg/model/push_context.go

```diff
@@ -1683,11 +1683,10 @@ func newConsolidatedDestRules() *consolidatedDestRules {
 	}
 }
 
-// SetDestinationRules is updates internal structures using a set of configs.
+// updateDestinationRules updates internal structures using a set of configs.
 // Split out of DestinationRule expensive conversions, computed once per push.
-// This also allows tests to inject a config without having the mock.
-// This will not work properly for Sidecars, which will precompute their destination rules on init
-func (ps *PushContext) SetDestinationRules(configs []config.Config) {
+// This will not work properly for Sidecars, which precomputes destination rules on init.
+func (ps *PushContext) updateDestinationRules(configs []config.Config) {
```

Comment:
I donot think frequently renaming is friendly to developers, a good practice is only do this once the function name could not express what it is doing,

Reply:
Yeah. I agree. Just exploring this path.